tests/test_api/test_endpoints.py

```diff
@@ -3220,3 +3220,44 @@ def test_find_studies_with_using_cursor_pagination_and_no_results(api_user_clien
     assert res_body["meta"]["count"] == 0
     assert res_body["meta"]["next_cursor"] is None
     assert res_body["results"] == []
+
+
+def test_create_job(api_site_client):
+    """Test creating a new job via POST /jobs endpoint."""
+    mdb = get_mongo_db()
+    faker = Faker()
+    #  create fake jobs
+    job = faker.generate_jobs(1)[0]
+    try:
+        # Submit the job creation request
+        response = api_site_client.request(
+            "POST",
+            "/jobs",
+            job,
+        )
+        # Verify the response
+        assert response.status_code == 200
+        created_job = response.json()
+        
+        # Verify the job has required fields
```

Comment:
I'm not sure what "required" is being used to mean in this case.

```suggestion
        # Verify the job has the fields we expect
```